ethdk/src/ExternallyOwnedAccount/ExternallyOwnedAccount.ts

```diff
@@ -59,8 +59,7 @@ export default class ExternallyOwnedAccount implements Account {
     }
 
     return this.createAccountFromPrivateKey({
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      privateKey: privateKey!,
```

Comment:
We no longer need the `!` because I changed the return type of isNullOrUndefined() to `value is null | undefined `

Reply:
Nice! Didn't know you could achieve that by adding `value is null | undefined` - much cleaner